xbmc/cores/VideoPlayer/VideoPlayer.h

```diff
@@ -488,10 +488,9 @@ class CVideoPlayer : public IPlayer, public CThread, public IVideoPlayer,
     mutable CCriticalSection m_section;
     CSelectionStreams m_selectionStreams;
     std::vector<ProgramInfo> m_programs;
-    int m_program;
-    int m_videoIndex;
-    int m_audioIndex;
-    int m_subtitleIndex;
+    int m_videoIndex = -1;
+    int m_audioIndex = -1;
+    int m_subtitleIndex = -1;
```

Comment:
We switched our code style a while ago, can you brace-initialize member variables? Like:

```c++
int m_videoIndex{-1};
int m_audioIndex{-1};
int m_subtitleIndex{-1};
```

Reply:
No problem :relaxed: